docs/adr/0013-add-native-support-biblatex-software.md

```diff
@@ -0,0 +1,49 @@
+# \[Added Native Support for BibLatex-Sotware\]
+
+* Status: Proposed
+* Deciders: Oliver Kopp
+
+Technical Story: [6574-Adding support for biblatex-software](https://github.com/JabRef/jabref/issues/6574)
+
+## Context and Problem Statement
+
+JabRef does not right now have support for Biblatex-Software out of the box , users have to add custome entry type.
+With citing software becoming fairly comen , native support would be helpful.
+
+
+## Decision Drivers
+
+* The new entry types definitions should be added to the Select Entry Pane and be separated by a divider
```

Comment:
I think, this line should be removed. It is not a "global driver", but one implementation option.

The other point is OK. - Even if it is a single one.